cpp/src/io/parquet/compact_protocol_reader.cpp

```diff
@@ -311,7 +313,6 @@ struct parquet_field_enum_list : public parquet_field_list<Enum, FieldType::I32>
     : parquet_field_list<Enum, FieldType::I32>(f, v)
   {
     auto const read_value = [&val = v](uint32_t i, CompactProtocolReader* cpr) {
-      CUDF_EXPECTS(i < val.size(), "Index out of bounds");
```

Comment:
redundant check, with the way the function is called

Reply:
I think this was added due to suppress a false positive from security vulnerability check tool